data/sql/updates/pending_db_world/rev_1683530792489127900.sql

```diff
@@ -0,0 +1,12 @@
+--
+UPDATE `creature_template` SET `minlevel`=63, `maxlevel`=63, `AIName`='SmartAI' WHERE  `entry`=16383;
+
+DELETE FROM `creature_template_addon` WHERE entry=16383;
+INSERT INTO `creature_template_addon` (`entry`, `auras`) VALUES (16383, '28330');
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=16383;
+INSERT INTO `smart_scripts` (`entryorguid`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `target_type`, `comment`) VALUES 
+(16383, 2000, 2000, 30000, 45000, 11, 28314, 2, 'Flameshocker - In Combat - Cast Victim `Flameshocker\'s Touch`');
+INSERT INTO `smart_scripts` (`entryorguid`, `id`, `event_type`, `action_type`, `action_param1`, `action_param2`, `target_type`, `comment`) VALUES 
+(16383, 1, 6, 11, 28323, 2, 1, 'Flameshocker - On Death - Cast Victim `Flameshocker\'s Revenge`');
+
```

Comment:
1. When deleting, always specify the source_type as well
2. Also this insert is a bit... well, try adding the Keira output instead :P 
```suggestion
DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 16383);
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(16383, 0, 0, 0, 0, 0, 100, 0, 5000, 10000, 10000, 20000, 0, 11, 28314, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Flameshocker - In Combat - Cast \'Flameshocker`s Touch\''),
(16383, 0, 1, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 11, 28323, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Flameshocker - On Just Died - Cast \'Flameshocker`s Revenge\'');
```

Reply:
`smart_scripts` insert needed adjusting but not `creature_template_addon`? Also, might want to stick whatever the issue is with that insert in the wiki.